adapters/tlsio_wolfssl.c

```diff
@@ -768,7 +768,7 @@ void tlsio_wolfssl_destroy(CONCRETE_IO_HANDLE tls_io)
 
 int tlsio_wolfssl_open(CONCRETE_IO_HANDLE tls_io, ON_IO_OPEN_COMPLETE on_io_open_complete, void* on_io_open_complete_context, ON_BYTES_RECEIVED on_bytes_received, void* on_bytes_received_context, ON_IO_ERROR on_io_error, void* on_io_error_context)
 {
-    int result;
+    int result = 0;
```

Comment:
Lets not give default value and let each branch set this value.

Reply:
FYI, these files are not consistent in their approach.  mbedtls does start with int result = 0 at this point. I understand the value of not starting with result = 0, but I also think we need to be consistent in our SDK.